bin/problem.py

```diff
@@ -333,9 +333,7 @@ def _determine_statement_languages(self):
         If problem.yaml's name key is a string, convert into dict; assume `en` as default language.
         """
         yamllangs = set(self.settings.name)
-        texlangs = set(
-            path.suffixes[0][1:] for path in glob(self.path, "problem_statement/problem.*.tex")
-        )
+        texlangs = set(path.suffixes[0][1:] for path in glob(self.path, "statement/problem.*.tex"))
```

Comment:
It's probably best to move this glob string to the enum as well, for consistency and single-source-of-truth?

Reply:
problem can't import latex because of circular imports ): any idea?

Reply:
Ah :sweat_smile: I ran into this before. I have two ideas:
- In `latex.py`, we could move `import problem` into an `if TYPE_CHECKING:` block
- We could move the `PdfType` enum to `util.py`